src/xenia/kernel/xboxkrnl/xboxkrnl_module.cc

```diff
@@ -186,6 +186,10 @@ XboxkrnlModule::XboxkrnlModule(Emulator* emulator, KernelState* kernel_state)
                                        ordinals::XexExecutableModuleHandle,
                                        ppXexExecutableModuleHandle);
 
+  uint32_t ppExLoadedImangeName = memory_->SystemHeapAlloc(FILENAME_MAX);
```

Comment:
Apart from the FILENAME_MAX issue (even though we allocate this memory by ourselves, games using this string likely don't expect it to be longer than MAX_PATH), there's a typo ("imange").